aptos-move/framework/aptos-framework/sources/transaction_context.move

```diff
@@ -46,6 +51,36 @@ module aptos_framework::transaction_context {
         auid.unique_address
     }
 
+    /// Return the sender's address for the current transaction.
+    /// This function aborts if called outside of the transaction prologue, execution, or epilogue phases.
+    native fun sender_internal(): address;
+    public fun sender(): address {
+        sender_internal()
+    }
+
+    /// Return the list of the secondary signers for the current transaction.
+    /// If the current transaction has no secondary signers, this function returns an empty vector.
+    /// This function aborts if called outside of the transaction prologue, execution, or epilogue phases.
+    native fun secondary_signers_internal(): vector<address>;
+    public fun secondary_signers(): vector<address> {
+        secondary_signers_internal()
+    }
+
+    /// Return some fee payer address if a separate fee payer is specified for the current transaction.
+    /// If no separate fee payer is specified, this function returns `None`.
+    /// This function aborts if called outside of the transaction prologue, execution, or epilogue phases.
+    native fun fee_payer_opt_internal(): Option<address>;
+    public fun fee_payer_opt(): Option<address> {
+        fee_payer_opt_internal()
+    }
+
+    /// Return the max gas amount specified for the current transaction.
```

Comment:
Add comments to explain what units this is in and whether this includes gas price